website/docs/components/stepper/list/partials/code/how-to-use.md

```diff
@@ -0,0 +1,75 @@
+## How to use this component
+
+```handlebars
+<Hds::Stepper::List @ariaLabel="Basic usage" as |S|>
+  <S.Step>
+    <:title>One</:title>
+  </S.Step>
+  <S.Step>
+    <:title>Two</:title>
+  </S.Step>
+  <S.Step>
+    <:title>Three</:title>
+  </S.Step>
+</Hds::Stepper::List>
+```
+
+### Indicating status
+
+To change the status of a Step, set the `@status` argument in the `[S].Step` contextual component. The default status value is `incomplete`.
+
+```handlebars
+<Hds::Stepper::List @ariaLabel="Indicating status" as |S|>
+  <S.Step @status="complete">
+    <:title>Completed step</:title>
+  </S.Step>
+  <S.Step @status="progress">
+    <:title>Current step</:title>
+  </S.Step>
+  <S.Step>
+    <:title>Upcoming step</:title>
+  </S.Step>
+</Hds::Stepper::List>
+```
+
+A `processing` status is used instead of `progress` to indicate an ongoing process in the background, such as a user's data being submitted.
```

Comment:
```suggestion
A `processing` status is used instead of `progress` to indicate an ongoing process in the background, such as the user's data being submitted.
```

Only one specific user is using the Stepper at a time and it's their specific data which is being submitted so I think in explanations like this it makes more sense to refer to "**the** user".